### ISSUE TITLE
Certain PDF's cause really wacky gcode.

### ISSUE THREAD

**Issue by zimmer62** [wreality/laser-gcode-app#16]
I can give you the pdf's but I think it has to do with the version, Acrobat 8 (PDF 1.7)

Everything was duplicated, turned 15 degreesish

**Comment by wreality** [comment]
Yeah, I'd love to see the pdfs on this one.  Likely it'll be an issue with pstoedit so I'm not sure how much can be done about it until the switch away from pstoedit happens.

**Comment by zimmer62** [comment]
I don't remember this one, and I haven't seen it happen again.  I'd love to figure out the solid color problems on my process though.
